manager_config.yaml

```diff
@@ -0,0 +1,66 @@
+manager:
+  private_ip:
+  public_ip:
+  hostname: <manager host name, e.g. manager2>
+  security:
+    ssl_enabled: true
+    admin_password: admin
+
+rabbitmq:
+  ca_path:
+  cluster_members:
+    <first rabbitmq instance-name>:
+      networks:
+        default: <first rabbitmq private-ip>
+    <second rabbitmq instance-name>:
+      networks:
+        default: <first rabbitmq private-ip>
+    <third rabbitmq instance-name>:
+      networks:
+        default: <first rabbitmq private-ip>
+
+postgresql_server:
+  ssl_enabled: true
+  ca_path:
+
+  postgres_password: 'somesecretvalue123!'
+  cluster:
+    nodes:
+      <first postgresql instance-name>:
+        ip:
+      <second postgresql instance-name>:
+        ip:
+      <third postgresql instance-name>:
+        ip:
+
+postgresql_client:
+  ssl_enabled: true
+  server_password: 'somesecretvalue123!'
+  ssl_client_verification: true
+
+validations:
+  minimum_required_total_physical_memory_in_mb: 1024
+
+ssl_inputs:
+  ca_cert_path:
+  internal_cert_path:
+  internal_key_path:
+  postgresql_client_cert_path:
+  postgresql_client_key_path:
+  external_cert_path:
+  external_key_path:
+  external_ca_cert_path:
+
+
+prometheus:
+  credentials:
+    username: 'monitoringusername'
+    password: 'longyeteasytorememberstringasapassword'
+
+  cert_path:
+  key_path:
+  ca_path:
+
```

Comment:
OK, general comment regarding these yaml files as I might've missed something in the rest of the review: Are these just example files or are they templates? I'm not sure examples are needed (if we want to generate modifiable example config then that should probably be an argument that does everything except run cfy_manager install), and if they're templates then let's make them actual templates, e.g. jinja2 or similar.

Reply:
Right, jinja is a great idea! I was wondering what should be the best approach, and I guess jinja template is the best. We'll modify it now.